pipeline-runner/R/qc-7-embed_and_cluster.R

```diff
@@ -4,6 +4,18 @@
 # done also in this step.
 
 embed_and_cluster <- function(scdata, config, sample_id, task_name = 'configureEmbedding') {
+  message("starting clusters")
+  req <- list()
+  type <- config$clusteringSettings$method
```

Comment:
Change req and everything to just passing parameters that ar eneeded

Reply:
naming suggestion: change `type` to `method`, because it is more accurate